src/main/Company.java

```diff
@@ -0,0 +1,71 @@
+
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Company  {
+
+
+    private  CEO ceo;
+    private List<Manager> managerList = new ArrayList<>();
+    private  String name;
+
+    public Company(String name, CEO ceo){
+        this.name = name;
+        this.ceo = ceo;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void display() {
+
+        System.out.println(this.getName()+ this.ceo.getName());
+
+        for (Manager m : managerList) {
+            System.out.println("  " + m.getName() + " " +m.getSalary() +" "+ m.isSatisfied()); // wyswietl
+            for (Employee e : m.getEmployeeList()) {
+                System.out.println( "     " + e.getName()+ " " +e.getSalary()+ " " + e.isSatisfied()); // wyswietl pracownika
+            }
+        }
+    }
+
+
+    public void fire(){
+        this.ceo = null;
+
+        }
+
+     public void hire(CEO ceo) {
+
+        if(ceo == null){
+            this.ceo =ceo;
+     }
+
+
+//    public void addEmoployee(AbstractEmployee employee,String name){
+  //      this.employeeList.add(employee,name);
```

Comment:
Nie zostawiamy takiego pokomentowanego kodu jest to code smell i jest bardzo nieładne.